src/model/APNSRequest.php

```diff
@@ -48,6 +74,10 @@ public function get_url_for_configuration( APNSConfiguration $configuration ): s
 		return $configuration->get_endpoint() . $this->token;
 	}
 
+	public function get_userdata(): object {
```

Comment:
Feel free to nitpick this name – I'm not in love with it, but I also couldn't come up with something better offhand 😅